src/app/shared/user.service.ts

```diff
@@ -52,18 +54,20 @@ export class UserService {
         .get(this.userUrl, {headers: this.headers})
         .toPromise()
         .then(response => {
-          let userData = response.json().data as User;
+          let userData = response.json().data;
           // The reference of this.userData is
           // being used in Header
           // So updating the value like that
           this.userData.attributes = {
             fullName: userData.attributes.fullName,
```

Comment:
Object.assign(this.userData.attributes, userData.attributes) might be a better long term plan for copying over attributes